doc/api/util.md

```diff
@@ -755,6 +775,55 @@ added: v8.0.0
 * {symbol} that can be used to declare custom promisified variants of functions,
 see [Custom promisified functions][].
 
+## util.setReportEvents(events)
+<!-- YAML
+added: v11.0.0
+-->
+
+* `events` {string}
+
+Runtime configuration of node report data capture. The string can be a comma-
+separated list of one or more of:
```

Comment:
The examples show this as a `+` separated list, not commas. In general, an array of individual values would be better.

Reply:
This is fixed and is documented.